### PR TITLE
Added example for show_axes_all

### PR DESCRIPTION
**Pull request by mbebic**
For issue #1629

Added example for pyvista.Plotter.show_axes_all().

### CODE REVIEW

**Comment by tkoyama010** [comment]
@mbebic Thank you for your first contribution. I think you can fix this using [style-checking of PyVista project](https://docs.pyvista.org/extras/developer_notes.html#style-checking).

File: pyvista/plotting/plotting.py [hunk 1]
```diff
@@ -1564,7 +1564,31 @@ def hide_axes_all(self):
             renderer.hide_axes()
 
     def show_axes_all(self):
-        """Show the axes orientation widget in all renderers."""
+        """Show the axes orientation widget in all renderers.
+        
+        Example
```
Comment (MatthewFlamm) [inline pyvista/plotting/plotting.py:1569]:
```suggestion
        Examples
```

**Review by MatthewFlamm — APPROVED** [review]
Other than comment, LGTM

File: pyvista/plotting/plotting.py [hunk 2]
```diff
@@ -1564,7 +1564,31 @@ def hide_axes_all(self):
             renderer.hide_axes()
 
     def show_axes_all(self):
-        """Show the axes orientation widget in all renderers."""
+        """Show the axes orientation widget in all renderers.
+
```
Comment (MatthewFlamm) [inline pyvista/plotting/plotting.py:1568]:
```suggestion

```

File: pyvista/plotting/plotting.py [hunk 3]
```diff
@@ -1564,7 +1564,30 @@ def hide_axes_all(self):
             renderer.hide_axes()
 
     def show_axes_all(self):
-        """Show the axes orientation widget in all renderers."""
+        """Show the axes orientation widget in all renderers.
+        Examples
```
Comment (MatthewFlamm) [inline pyvista/plotting/plotting.py:1568]:
```suggestion

        Examples
```

**Pull request merged**